site/content/usage/experimental/01-gitops.md

```diff
@@ -17,35 +17,35 @@ and simplify application deployments and operations tasks to Kubernetes.
 
 ### Installing Flux
 
-> This is an experimental feature. To enable it, set the environment variable `EKSCTL_EXPERIMENTAL=true`. 
+> This is an experimental feature. To enable it, set the environment variable `EKSCTL_EXPERIMENTAL=true`.
 > Experimental features are not stable and their command name and flags may change.
 
-Installing Flux on the cluster is the first step towards a gitops workflow. To install it, you need a Git repository 
+Installing Flux on the cluster is the first step towards a gitops workflow. To install it, you need a Git repository
 and an existing EKS cluster. Then run the following command:
 
 ```console
-EKSCTL_EXPERIMENTAL=true eksctl install flux --name <cluster_name> --region <region> --git-url=<git_repo> --git-email=<git_user_email>
+EKSCTL_EXPERIMENTAL=true eksctl install flux --cluster <cluster_name> --region <region> --git-url=<git_repo> --git-email=<git_user_email>
```

Comment:
nit: you could make flags more consistent in this page - either with equals sign or without

Reply:
Good point, even the `--region` option is inconsistent.